lib/modules/manager/argocd/extract.ts

```diff
@@ -36,6 +36,7 @@ export function extractPackageFile(
     definitions = parseYaml(content, null, {
       customSchema: ApplicationDefinition,
       failureBehaviour: 'filter',
+      removeTemplates: true,
```

Comment:
@viceice maybe this should be the default?

Reply:
not sure, but probably yes.